LiskHQ/lisk-desktop
# Bugfix account balance calculation

### What was the problem?

This PR resolves #5349, #5347

### How was it solved?

- Fix account balance and usable balance bug by check against available balance and amount to spend including fee
- Fix transaction incoming and outgoing arrow

### How was it tested?

- Check wallet for incoming and outgoing transaction
- Send entire balance (considering fees)
